modules/packages/LinearAlgebra.chpl

```diff
@@ -2063,6 +2319,176 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
   return C;
 }
 
+/*
+  Matrix exponential using Pade approximation. This method returns N*N matrix which
+  is Matrix exponential of `A`
+
+  :arg A: Expects an N*N square matrix.
+  :type A: 2D Array
```

Comment:
I think this is missing backticks, did `make docs` work for you?  I got this message:
```
Warning, treated as error:
/Users/lydia/Documents/Repositories/chapel/doc/rst/modules/packages/LinearAlgebra.rst:801:chpl:chplref reference target not found: 2D Array
```

Reply:
We're so close!  Almost there :)